workshops/tunes/README.md

```diff
@@ -0,0 +1,203 @@
+
+At the top of `index.html` we reference our stylesheets and metadata. To make stylinging super easy I've also added a [Github Markdown CSS file](https://github.com/sindresorhus/github-markdown-css).
+
+<img src="https://cloud-f8397euam.vercel.app/screenshot_2020-09-17_at_7.06.42_pm.png" width="380" alt="Our starter site">
+
+```html
+<body class="markdown-body">
+  <div>
+    <h1>Piano on the Web</h1>
+    <p>Hey there! This site allows you to make music like you would on any piano! Let's make some noise!</p>
+    <p>Click the <kbd>S</kbd> key to make a middle C (or C4), then <kbd>D</kbd> to make D4 and so on up until B4 with the <kbd>K</kbd> key.</p>
+  </div>
+  <div id="piano">
+    <span class="key" id="C"><strong>C</strong></span>
+    <span class="key" id="D"><strong>D</strong></span>
+    <span class="key" id="E"><strong>E</strong></span>
+    <span class="key" id="F"><strong>F</strong></span>
+    <span class="key" id="G"><strong>G</strong></span>
+    <span class="key" id="A"><strong>A</strong></span>
+    <span class="key" id="B"><strong>B</strong></span>
+  </div> 
+  <script src="https://cdnjs.cloudflare.com/ajax/libs/tone/14.7.52/Tone.js"></script>
+  <script src="script.js"></script>
+</body>
+```
+
+I'll quickly break this down. The class applied to the `<body>` tag ensures that the Github Markdown styles are applied. Then in the first `<div>` tag we provide a basic introduction to the site. Then in the second `div` there are a `span` element for each key we will be using. Then at the end we add all of our Javascript scripts. Next up let me breakdown the CSS file.
+
+```css
+body{
+  width: 75vw;
+  margin: auto;
+}
+
+#piano{
+  height: 400px;
+}
+
+.key{
+  height: 400px;
+  width: 14.2857142857%;
+  float: left;
+  border: 5px solid black;
+  border-right: none;
+  background: white;
+  text-align: center;
+  padding-top: 300px;
+}
+
+#B{
+  border-right: 5px solid black;
+}
+```
+
+Firstly, the `body` tag is set to have a width and centered using `margin: auto;`. Secondly, we set the `#piano` div to have a height of 400px. As there is very little content in the keys, this will make sure they are big and look realistic. Next we make each key have the correct height, have a width of one seventh of the div, it has a border and a background of white. Lastly, the B key will have a right border as it comes at the end.
+
+I hope you understand the starter, now you may be getting bored so let's get making!
+
+# Detecting a key press
+
+To play a note in our project we will have the user press a key on their keyboard. Javascript has an event for this: `document.onkeydown`. This is fired when ever a key is pressed down.
+
+For example, in the below snippet:
+
+```javascript
+document.onkeydown = function(e){
+	console.log('Key pressed!')
```

Comment:
Indentation should be the same as repl.it's so please make it 2 spaces everywhere.